pkg/server/server.go

```diff
@@ -384,6 +386,12 @@ func (s *Server) ListObjects(ctx context.Context, req *openfgav1.ListObjectsRequ
 		methodName,
 	).Observe(queryCount)
 
+	requestDurationByQueryHistogram.WithLabelValues(
+		openfgav1.OpenFGAService_ServiceDesc.ServiceName,
+		methodName,
+		utils.Bucketize(uint(*result.ResolutionMetadata.QueryCount), s.requestDurationByQueryHistogramBuckets),
+	).Observe(float64(time.Since(start).Milliseconds()))
```

Comment:
**Suggestion (non-blocking)**: I think, we should define the OpenFGA Desc `serviceName`. As we are re-using it again & again in the whole file (9 occurrences -- not all, but we can optimize for some of them), and we can define it as a constant like we did for `methodName`. Maybe something to refactor to later on, something like:

```go
// Deinfing & extracting service name for clarity 
serviceName := openfgav1.OpenFGAService_ServiceDesc.ServiceName

// Calculate the duration in milliseconds -- Optional suggestion for better readability.
durationMs := float64(time.Since(start).Milliseconds())

requestDurationByQueryHistogram.WithLabelValues(
    serviceName,
    methodName,
    utils.Bucketize(uint(*result.ResolutionMetadata.QueryCount), s.requestDurationByQueryHistogramBuckets),
).Observe(durationMs)

```

Reply:
Thanks for the suggestion @Siddhant-K-code .  Added common serviceName as suggested